seqeval/callbacks.py

```diff
@@ -111,7 +114,8 @@ def score(self, y_true, y_pred):
         """
         score = f1_score(y_true, y_pred)
         print(' - f1: {:04.2f}'.format(score * 100))
-        print(classification_report(y_true, y_pred, digits=4))
+        if self.digits:
+            print(classification_report(y_true, y_pred, digits=self.digits))
```

Comment:
Hello @alexeyr 

I think If `self.digits` is `None`, this line raises an Exception `TypeError: '>' not supported between instances of 'NoneType' and 'int'`.

So, If `self.digits` is `None`, we need to pass a default value to classification_report:

```python
if self.digits:
    print(classification_report(y_true, y_pred, digits=self.digits))
else:
   print(classification_report(y_true, y_pred, digits=4))
```

or, we can set default value in `__init__` method:

```python
self.digits = digits or 4
```

I think the latter is a smart solution. Thanks!

Reply:
if `self.digits` is `None`, it shouldn't print anything, that was the point of allowing `None`. I could write `if self.digits is not None` to make it more explicit.